### ISSUE TITLE
interruptInputListener() on blur breaks mouse dragging for GrabDragInteractions

### ISSUE THREAD

**Issue by zepumph** [phetsims/scenery-phet#643]
Over in https://github.com/phetsims/faradays-law/issues/201, @jbphet reported that there is a problem when you try to drag with a mouse/touch after/during use of keyboard with a GrabDragInteraction.

Steps to reproduce:
1. In Friction/Gravity Force Lab/ Faradays Law
2. Highlight the grab drag interaction (book/ruler/magnet)
3. Use a mouse and try to drag it while it is focused with a keyboard (see https://github.com/phetsims/faradays-law/issues/201#issuecomment-728540347 for video)

When you try to drag an item that is a GrabDragInteraction, the blurring of the PDOM element causes Node.interruptInputListeners to be called. In most cases, the drag listener is on the same Node as the GrabDragInteraction, so this interrupts the drag listener too. 

In BASE, which also uses GrabDragInteraction, this isn't an issue because the GrabDragInteraction is added to a child of the the Node which gets the DragListener, thus, the DragListener never gets interrupted. I feel like this is a workaround at best, and buggy inconsistency at worst.

@jessegreenberg, I'd like to discuss how best to fix this issue. This patch solves it, but I wasn't sure this is actually the way to handle things. 

```diff
Index: js/accessibility/pdom/ParallelDOM.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- js/accessibility/pdom/ParallelDOM.js	(revision 7794ca08ebeda1d1037b05f44df3815e989b9693)
+++ js/accessibility/pdom/ParallelDOM.js	(date 1605656780815)
@@ -122,8 +122,8 @@
  * @author Michael Kauzmann (PhET Interactive Simulations)
  */
 
-import ValidatorDef from '../../../../axon/js/ValidatorDef.js';
 import validate from '../../../../axon/js/validate.js';
+import ValidatorDef from '../../../../axon/js/ValidatorDef.js';
 import Shape from '../../../../kite/js/Shape.js';
 import arrayDifference from '../../../../phet-core/js/arrayDifference.js';
 import extend from '../../../../phet-core/js/extend.js';
@@ -555,7 +555,7 @@
           const peer = this._accessibleInstances[ 0 ].peer;
           assert && assert( peer, 'must have a peer to blur' );
           peer.blur();
-          this.interruptInput(); // interrupt any listeners that attached to this Node
+          // this.interruptInput(); // interrupt any listeners that attached to this Node
         }
       },
 

```

I can't currently think of a way around this issue. Marking for meeting with priority so we get to it next week.

**Comment by zepumph** [comment]
Please note that this is not just for GrabDragInteraction. It is for any draggable and keyboard navigable Node. For example I can reproduce with Ratio and Proportion hands (which are `input` sliders in the PDOM).

**Comment by zepumph** [comment]
@jessegreenberg and I discussed this today. We discussed what it would look like to remove the above line, and to not interrupt on blur.

KeyboardDragListener supports interrupt, so we would want to be careful about cases that use it to make sure they are interrupted when blurs happen on their Nodes. GrabDragInteraction is already handling this for us:  https://github.com/phetsims/scenery-phet/blob/578b1cb09f0fffc9c3145a8f4da6346a0e5fea58/js/accessibility/GrabDragInteraction.js#L529. 

This would be  a change in philosophy, in which blur doesn't automatically interrupt,,, but instead you choose on a case by case basis. We then looked through all usages of `keydown`, which seems to be the only a11y event that has a longer timeframe than a single listener.

`MagnetAutoSlideKeyboardListener` doesn't have an interrupt, but after that above patch was applied, @jessegreenberg and I were surprised that we didn't see the magnet continue to animate after it was blurred. We witnessed it stop when we tabbed away. We see no `blur`, or `interrupt` in any files associated with the type. Our best guess is that this is because of a custom case using `isAnimatingProperty`, and has nothing to do with the input system. (https://github.com/phetsims/faradays-law/blob/3be85930b0e605152df7a7119fde6d3b8d95a7e5/js/faradays-law/view/MagnetAutoSlideKeyboardListener.js#L176-L180)

From this we feel good removing this interrupt in `blur()`.

**Comment by zepumph** [comment]
Local fuzzing is passing. Over to @jbphet to review for FL and to comment, if he desires, on MagnetAutoSliderKeyboardListener.

**Comment by jbphet** [comment]
The behavior is fixed for Faraday's Law, though the drop and pickup sounds both play in the use case described.  I'll discuss with the design team whether that's okay, since I suspect it might be tricky to prevent.  That discussion will occur in https://github.com/phetsims/faradays-law/issues/201.

The change itself, which involves the removal of a single line of code, doesn't strike me as problematic, and @zepumph's explanation above and testing seems to support it.  I don't know enough about input events and the interrelationship between keyboard and pointer events to know whether there are any broader implications.  @zepumph - if you're okay with that, I think this can be closed.
